scripts/asbcli/asbcli

```diff
@@ -460,14 +468,16 @@ class App(object):
 # 24   -v \
 # 25   http://cap.example.com:1338/v2/service_instances/$instanceUUID
 ############################################################
-    def _req_provision(self, service_id, answers):
+    def _req_provision(self, service_id, answers, namespace):
```

Comment:
Absolutely. The broker expects a namespace on every op as of TP1. I'd really like to keep asbcli up-to-date if its still helpful for people, so this is good stuff. I started to patch this myself, but I'm hoping to catch up with @jmrodri to talk about what we need to do to keep asbcli alive (much to my chagrin :P)